src/libraries/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/AesCcm.Browser.cs

```diff
@@ -0,0 +1,10 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Would it make sense to rename these no-op files to have `.PlatformNotSupported.cs` suffix? It's likely that we will need to reuse them on mobile.

Reply:
On second thought the mobile platforms don't run the PNSE generator now and if they did they'd need to run in opt-in fashion instead of opt-out fashion to generate specific types.

Reply:
Renaming the file should be fine. Even if right now the only thing that uses it is _Browser_, it at least provides the scaffolding for us to hook more targets up in the future.

Reply:
I'd go with `NotSupported`, or `Unsupported` since it's the same as `PlatformNotSupported` but doesn't mess with `ls` columnar output as much :smile: